test/images/agnhost/VERSION

```diff
@@ -1 +1 @@
-2.4
+2.5
```

Comment:
while you're modifying agnhost/webhook, should you add a port arg so we can exercise webhooks listening on non-443 ports? should be ~3 lines in webhook/main.go

Reply:
Yes, absolutely. Adding now.